packages/frontend/src/features/metricsCatalog/components/MetricPeekModal.tsx

```diff
@@ -149,6 +180,41 @@ export const MetricPeekModal: FC<Props> = ({ opened, onClose }) => {
                 >
                     <Stack p="xl" bg="offWhite.0" miw={360}>
                         <Stack spacing="xl">
+                            {metricQuery.data?.availableTimeDimensions && (
```

Comment:
user can only pick from date dims if there are available time dimensions - and these are only defined **if there is no default time dimension**